index.html

```diff
@@ -126,187 +868,623 @@ <h3>
+                <a href="https://streams.spec.whatwg.org/#desired-size-to-fill-a-streams-internal-queue">desired size</a>
+                of [=this=].{{UDPSocket/[[readable]]}}'s
+                <a href="https://streams.spec.whatwg.org/#internal-queues">internal queue</a>.
+              <li>
+                Run the following steps in parallel:
+                <ol>
+                  <li>
+                    Invoke the operating system to provide us |desiredSize| UDP packets from the socket.
+                  <li>
+                    [=Queue a global task=] on the [=relevant global object=] of
+                    [=this=] using the [=UDP socket task source=] to run the
+                    following steps for each received packet:
+                    <ol>
+                      <li>
+                        If no errors were encountered run the following steps:
+                        <ol>
+                          <li>Let |bytes| be a [=byte sequence=] containing the packet data. 
+                          <li>
+                            Let |buffer| be a [=new=] {{ArrayBuffer}} created from |bytes|.
+                          <li>
+                            Let |chunk| be a [=new=] {{Uint8Array}} view over |buffer|,
+                            who's length is the length of |bytes|.
+                          <li> 
+                            Let |message:UDPMessage| be a new {{UDPMessage}}.
+                          <li>
+                            Set |message|["{{UDPMessage/data}}"] to |chunk|.
+                          <!--
+                          <li>
+                            Infer the origins of the received UDP packet and populate
+                            |message|["{{UDPMessage/remoteAddress}}"] and |message|["{{UDPMessage/remotePort}}"] accordingly. 
+                          -->
+                          <li>
+                            Invoke [=ReadableStream/enqueue=] on
+                            [=this=].{{UDPSocket/[[readable]]}} with |message|.
+                        </ol>
+                      <li>
+                        If an operating system error was encountered, invoke
+                        [=ReadableStream/error=] on [=this=].{{UDPSocket/[[readable]]}} with
+                        a "{{NetworkError}}" {{DOMException}}, discard other packets and invoke the steps
+                        to [=handle closing the UDP readable stream=].
+                    </ol>
+                </ol>
+              <li>
+                Return [=a promise resolved with=] `undefined`.
+            </ol>
+          <li>
+            Let |cancelAlgorithm| be the following steps:
+            <ol>
+              <li>
+                Let |promise:Promise| be [=a new Promise=].
+              <li>
+                Run the following steps [=in parallel=]:
+                <ol>
+                  <li>
+                    Invoke the operating system to close the socket.
```

Comment:
As with TCP, I think this should just be a no-op.

Reply:
Fixed